src/desktop/camera.rs

```diff
@@ -115,81 +115,135 @@ impl<'a> CameraProxy<'a> {
     }
 }
 
-/// A helper to get the PipeWire Node ID to use with the camera file descriptor
-/// returned by [`CameraProxy::open_pipe_wire_remote`].
+#[cfg(feature = "pipewire")]
+pub struct Stream {
+    pub node_id: u32,
+    pub properties: HashMap<String, String>,
```

Comment:
should be private and have getters like other types

Reply:
Done. Added helpers for `nick`, `description` and `name`. It would be interesting to check whether all camera stream present these properties and unwrap_or_default them instead of returning an Option.